src/fragments/dropdowns.mdx

```diff
@@ -0,0 +1,15 @@
+The Dropdown component provides state and accessibility through the context API to its consumers while enabling
+customization.
+
+**Placement / Positioning**: Internally, the Dropdown component uses [PopperJS](https://popper.js.org/docs/v2/)
+for positioning. Use the `popperModifiers` prop of the Menu component to customize the placement, boundaries,
+overflows etc.
+
+**Dropdown State**: The [Downshift](https://github.com/downshift-js/downshift) `render-prop` library handles the
+keyboard and accessibility logic. Provide other customizations directly to the Downshift provider via the
+`downshiftProps` prop. See the [stateReducer](https://github.com/downshift-js/downshift#statereducer) pattern
+for an example of a common customization strategy.
+
+**Server Side Rendering**: Server side rendering might require configuring specific Downshift settings. The
+package mentioned re-exports the Downshift `resetIdCounter` utility, resetting the internal id counter which
```

Comment:
Would you specifically name the package even if it's repetitive? I had a moment of confusion what package.

Reply:
Updated!